### ISSUE TITLE
index.html fails Accessibility Test

### ISSUE THREAD

**Issue by PeterQHo** [PeterQHo/github-issue-templates#5]
**Describe the bug**
The index.html fails the Accessibility Test and is at risk of an accessibility lawsuit.

**To Reproduce**
1. Go to 'https://www.accessibilitychecker.org/'
2. Paste in  'https://peterqho.github.io/github-issue-templates/'
3. Click Check website

**Expected behavior**
Accessibility Test should be compliant

**Screenshots**
Click on the following link to view the current Accessibility Test errors:

https://www.accessibilitychecker.org/audit/?website=https%3A%2F%2Fpeterqho.github.io%2Fgithub-issue-templates%2Findex.html&flag=us 

**Desktop (please complete the following information):**
N/A

**Smartphone (please complete the following information):**
N/A 

**Additional context**
Use the Accessibility Test link to see if there are any issues or errors
